repositories/src/main/scala/cromwell/pipeline/datastorage/dto/ProjectConfiguration.scala

```diff
@@ -6,13 +6,19 @@ import ProjectFile.pathFormat
 import org.mongodb.scala.bson.collection.immutable.Document
 import play.api.libs.json.{ Json, OFormat }
 
-case class ProjectFileConfiguration(path: Path, inputs: List[FileParameter])
+case class ProjectFileConfiguration(path: Path, inputs: Set[FileParameter]) {
+   override def equals(obj: Any): Boolean = obj match {
+     case o: ProjectFileConfiguration => this.path == o.path
+     case _ => false
+   }
+   override def hashCode(): Int = path.hashCode()
```

Comment:
I disagree with this definition.

If we have two configurations for the same file, it doesn't mean the are the same. The whole point of ProjectFile**Configuration** is to have the configurations. It doesn't make sense to not use them in `equals` / `hashcode`implementations.

Reply:
Removed this overriding